### PR TITLE
OPSEXP-139 Document custom images overrides

### PR DESCRIPTION
**Pull request by gionn**
OPSEXP-139

### CODE REVIEW

**Review by alxgomz — CHANGES_REQUESTED** [review]
Can you please add:
* a mention that custom images should ALWAYS be used as we provide some just for convenient but do not commit to publish security update for softwares or lib that are out of our control
* Add an example of using a custom image from an authenticated registry.

File: docs/helm/README.md [hunk 1]
```diff
@@ -12,17 +12,48 @@ The Community configuration will deploy the following system:
 
 ![Helm Deployment Community](./diagrams/helm-community.png)
 
-## Considerations
+## What to expect from the ACS Helm charts
```
Comment (vprovaggi) [inline docs/helm/README.md:15]:
```suggestion
## Overview
```

File: docs/helm/README.md [hunk 2]
```diff
@@ -12,17 +12,48 @@ The Community configuration will deploy the following system:
...
 
-Alfresco provides tested Helm charts as a "deployment template" for customers who want to take advantage of the container orchestration benefits of Kubernetes. These Helm charts are undergoing continual development and improvement, and should not be used "as is" for your production environments, but should help you save time and effort deploying Alfresco Content Services for your organisation.
+Alfresco provides tested Helm charts as a "deployment template" for customers
+who want to take advantage of the container orchestration benefits of
+Kubernetes. These Helm charts are undergoing continual development and
+improvement, and should not be used "as is" for your production environments,
+but should help you save time and effort deploying Alfresco Content Services for
+your organisation.
```
Comment (vprovaggi) [inline docs/helm/README.md:22]:
```suggestion
Alfresco provides tested Helm charts as a "template" to accelerate deployment and configuration for customers
who wants to take advantage of Kubernetes orchestration capabilities. Please remember that as stated in our [support policies](https://docs.alfresco.com/support/latest/policies/deployment/), similarly to other deployment artifacts, they are not meant to be used 'as-is' in production.

Our goal is to save you time and effort deploying Alfresco Content Services for
your organization. 
```

File: docs/helm/README.md [hunk 3]
```diff
@@ -12,17 +12,48 @@ The Community configuration will deploy the following system:
...
+charts](https://github.com/bitnami/charts) and do not configure any logging or
+monitoring. This design has been chosen so that they can be installed in a
+Kubernetes cluster without changes and are still flexible to be adapted to your
+actual environment.
 
-For your environment, you should use these charts as a starting point and modify them so that ACS integrates into your infrastructure. You typically want to remove the PostgreSQL container and connect the cs-repository directly to your database (might require [custom images](../docker-compose/examples/customisation-guidelines.md) to get the required JDBC driver in the container).
+For your environment, you should use these charts as a starting point and
+extends them so that ACS integrates into your infrastructure. You typically want
+to remove the PostgreSQL container and connect the cs-repository directly to
+your database.
```
Comment (vprovaggi) [inline docs/helm/README.md:36]:
```suggestion
You typically want
to remove the PostgreSQL container and connect the cs-repository directly to
your database.
```

File: docs/helm/README.md [hunk 4]
```diff
@@ -12,17 +12,48 @@ The Community configuration will deploy the following system:
...
+Alfresco provides tested Helm charts as a "deployment template" for customers
+who want to take advantage of the container orchestration benefits of
+Kubernetes. These Helm charts are undergoing continual development and
+improvement, and should not be used "as is" for your production environments,
+but should help you save time and effort deploying Alfresco Content Services for
+your organisation.
 
-Helm charts values contains secrets to be set. For production deployments please see the [Security](security.md) section.
+Helm charts values contains secrets to be set. For deployment in sensitive
+environments please see the [Security](security.md) section.
```
Comment (vprovaggi) [inline docs/helm/README.md:25]:
```suggestion
## Using the Helm charts

To set your secrets please see the [Security](security.md) section, especially for deployment in sensitive
environments
```

File: docs/helm/README.md [hunk 5]
```diff
@@ -12,17 +12,48 @@ The Community configuration will deploy the following system:
...
 
-The Helm charts in this repository provide a PostgreSQL database in a Docker container and do not configure any logging. This design has been chosen so that they can be installed in a Kubernetes cluster without changes and are still flexible to be adopted to your actual environment.
+The Helm charts in this repository provide a PostgreSQL database via [Bitnami
+charts](https://github.com/bitnami/charts) and do not configure any logging or
```
Comment (vprovaggi) [inline docs/helm/README.md:28]:
```suggestion
Out of the box we provide a PostgreSQL database via [Bitnami
charts](https://github.com/bitnami/charts) and do not configure any logging or
```

**Review by vprovaggi — COMMENTED** [review]
my opinion on making the documentation a bit easier to read

File: docs/helm/README.md [hunk 6]
```diff
@@ -12,17 +12,54 @@ The Community configuration will deploy the following system:
 
 ![Helm Deployment Community](./diagrams/helm-community.png)
 
-## Considerations
+## Overview
 
-Alfresco provides tested Helm charts as a "deployment template" for customers who want to take advantage of the container orchestration benefits of Kubernetes. These Helm charts are undergoing continual development and improvement, and should not be used "as is" for your production environments, but should help you save time and effort deploying Alfresco Content Services for your organisation.
+Alfresco provides tested Helm charts as a "template" to accelerate deployment
+and configuration for customers who wants to take advantage of Kubernetes
```
Comment (alxgomz) [inline docs/helm/README.md:18]:
```suggestion
and configuration for customers who want to take advantage of Kubernetes
```

**Review by alxgomz — APPROVED** [review]

**Pull request merged**